client/js/lounge.js

```diff
@@ -447,8 +425,49 @@ $(function() {
 			socket.emit("sessions:get");
 		}
 
+		if (target === "#help" || target === "#changelog") {
+			const now = Date.now();
+			// Don't check more than once a day
+			if (now - changelogRequestedAt > 86400 * 1000) {
```

Comment:
One day was arbitrary. We don't release often so that's reasonable IMO, and there are probably not many people running the app 24/7 at this point, yet.